addons/rootfs/usr/bin/supervisor_run

```diff
@@ -22,7 +22,6 @@ function run_supervisor() {
         -v /run/dbus:/run/dbus:ro \
         -v /run/udev:/run/udev:ro \
         -v /mnt/supervisor:/data:rw \
-        -v "$WORKSPACE_DIRECTORY":/data/addons/local:rw \
```

Comment:
Shouldn't make this the workspace to be underneath the Supervisor's data directory (`/data`) as well? :thinking: 

Maybe this is a bind mount propagation issue, did you try with `,bind-propagation=rshared` or similar :thinking: